src/client/NetworkingClient.cpp

```diff
@@ -95,27 +97,25 @@ void NetworkingClient::RealtimeClientRecv() {
     sf::Uint32 client_id;
     float x_pos;
     float y_pos;
+    float size;
 
     while (true) {
         sf::Packet packet;
         sf::IpAddress sender;
         unsigned short port;
         mRealtimeClient->receive(packet, sender, port);
-        std::cout << "recv: " << sender << " " << port << std::endl;
         // fetch state updates for now
         if (packet >> server_tick) {
-            mAcceptingPositionRead = false;
-            mPositions.clear();
-            while (packet >> client_id >> x_pos >> y_pos) {
-                std::cout << "Client: " << client_id << " has position x,y: " << x_pos << " " << y_pos << std::endl;
-                position p = {client_id, x_pos, y_pos};
-                mPositions.push_back(p);
+            mAcceptingNetworkGameObjectsRead = false;
+            mNetworkGameObjects.clear();
+            while (packet >> client_id >> x_pos >> y_pos >> size) {
+                network_game_object ngo = {client_id, x_pos, y_pos, size};
+                mNetworkGameObjects.push_back(ngo);
             }
             // Im not sure what kind of synchronization needs to happen here.
             // If this tick is the most up-to-date we've ever seen, maybe we set the game to it?
-            std::cout << "Client positions from server for tick: " << server_tick << std::endl;
             mCurrentTick = server_tick;
-            mAcceptingPositionRead = true;
+            mAcceptingNetworkGameObjectsRead = true;
```

Comment:
I wonder if it's important that the word `network` be in the name of this variable as well as in the name of the vector `mNetworkGameObjects`... at the end of the day once it's in that vector these are just state updates, do we really care they're from the network?

Reply:
I do think we can do better here wrt naming / abstraction, like why don't we have a GameObject which defines the << >> functions so it can be serialized and we don't have a distinction that way

Reply:
Im fine to commit since we're still early and it's not worth focusing on